### PR TITLE
doc: add vulns fixed in recent security release

### PR DESCRIPTION
**Pull request by mhdawson**

### CODE REVIEW

**Comment by mhdawson** [comment]
@RafaelGSS can you look at the validator it seems to be complaning in a way that does not make sense to me.

File: vuln/core/107.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "cve": ["CVE-2023-24807"],
```
Comment (RafaelGSS) [inline vuln/core/107.json:2]:
```suggestion
  "cve": ["CVE-2023-23920"],
```

**Review by RafaelGSS — CHANGES_REQUESTED** [review]
@mhdawson This should fix it: https://github.com/nodejs/security-wg/pull/882

However, you don't need to update the index.json file. We have an action that will perform this for you https://github.com/nodejs/security-wg/blob/main/.github/workflows/update-core-index.yml.

**Comment by RafaelGSS** [comment]
@mhdawson I will go ahead and fix it. Feel free to revert if needed.

**Review by RafaelGSS — APPROVED** [review]

**Pull request merged**